docs/RELEASE.md

```diff
@@ -0,0 +1,54 @@
+# Release process and check offs (client only)
+______
+since there is no server component at this time
+
+## Prepare release
+1. [ ]  User testing on mockups (design)
+1. [ ]  go/nogo on individual features (engineering)
+1. [ ]  create release tag for the selected features
+1. [ ]  all automated tests pass
```

Comment:
These should be gating items, not process.
They should always be running.

Reply:
As I said before, willfix